app/models/guide.rb

```diff
@@ -3,7 +3,6 @@
 class Guide < ActiveRecord::Base
   validates :content_id, presence: true, uniqueness: true
   validates :slug, presence: true
-  validates_associated :latest_edition
```

Comment:
Minor: we spoke about it offline, but it would be nice to notice why you've removed this in the commit note.